authui/src/authflowv2/components/screen-description.css

```diff
@@ -0,0 +1,24 @@
+@layer components {
+  :root {
+    --screen-description-font-family: var(--font-family-title-medium);
+    --screen-description-font-size: var(--font-size-title-medium);
+    --screen-description-line-height: var(--line-height-title-medium);
+    --screen-description-letter-spacing: var(--letter-spacing-title-medium);
+    --screen-description-font-weight: var(--font-weight-title-medium);
+    --screen-description-text-align: var(--alignment-content);
```

Comment:
Please check `authui/src/authflowv2/components/screen-title.css` for the text-align style of this component. They should be similar.